src/components/DetailViewHeader.jsx

```diff
@@ -110,20 +134,52 @@ export class DetailViewHeader extends React.PureComponent {
                 canManageUsers={true}
               />
             )}
-            <Button className={classes.exportButton} variant="contained" color="primary">
-              <a
-                target="_blank"
-                rel="noopener noreferrer"
-                href={`${terraUrl}/#import-data?url=${window.location.origin}&snapshotId=${of.id}&snapshotName=${of.name}&format=snapshot`}
-              >
-                Export to Workspace
-              </a>
-            </Button>
+            <hr className={classes.separator} />
+            <Typography variant="h6" className={classes.section}>
+              Export a copy of the snapshot metadata to an exisiting or new Terra workspace
+            </Typography>
+            {!isProcessing && !isDone && (
+              <TerraTooltip title="Exporting a snapshot to a workspace means that all members of your workspace will be able to have read only access to the tables and files in the snapshot">
+                <Button
+                  onClick={this.exportToWorkspaceCopy}
+                  className={classes.exportButton}
+                  variant="outlined"
+                  color="primary"
+                >
+                  Export snapshot
+                </Button>
+              </TerraTooltip>
+            )}
+            {isProcessing && !isDone && (
+              <Button className={classes.exportButton} variant="outlined" color="primary">
```

Comment:
You should also disable the button here to avoid double-clicking

Reply:
Ok - I've added something to disable it. Although, since I'm having to handle disabling it with redux, I don't think that we're disabling this before we hide the button altogether. So, I'm not sure this logic needed.